apps/default-app/src/providers/DynamicProvider.tsx

```diff
@@ -0,0 +1,22 @@
+import { ReactNode, useMemo } from "react";
+
+import { DynamicContext } from "./shared/dynamicContext";
+
+type DynamicProviderProps = {
+  children: ReactNode;
```

Comment:
not a big deal, but typically we don't import types from React, as they're global and therefore not required. same with testing utilities (`test()`, `expect()`)


```suggestion
  children: React.ReactNode;
```